webplugin/js/app/mck-sidebox-1.0.js

```diff
@@ -9718,6 +9717,39 @@ var userOverride = {
                 }
                 return '';
             };
+            _this.getMessageCreatedAtTime = function (createdAtTime) {
+                if (TIME_FORMAT_24_HOURS) {
+                    var messageTime = new Date(createdAtTime);
+                    var currentTime = new Date();
+                    function addZeroInHours() {
+                        if (messageTime.getHours() < 10) {
+                            return "0";
+                        }
+                        return "";
+                    }
+                    function addZeroInMinutes() {
+                        if (messageTime.getMinutes() < 10) {
+                            return "0";
+                        }
+                        return "";
+                    }
+                    if (currentTime.getDate() != messageTime.getDate() || currentTime.getMonth() != messageTime.getMonth() || currentTime.getFullYear() != messageTime.getFullYear()) {
+                        return messageTime.toLocaleString('en-US', {
+                            day: 'numeric',
+                            month: 'short'
```

Comment:
lot of repeated code, figure out smarter ways to do it with less lines of code without repetition

Reply:
I hope this will be better now. Changed the whole code.